extensions/functions_arithmetic.yaml

```diff
@@ -257,7 +258,7 @@ scalar_functions:
           on_domain_error:
             values: [ NAN, "NULL", ERROR ]
           on_division_by_zero:
-            values: [ LIMIT, NAN, "NULL", ERROR ]
+            values: [ NAN, "NULL", ERROR ]
```

Comment:
the `IEEE` option is only for fp32/fp32?

Reply:
Yes.  IEEE behavior is:

```
0 / 0 => NAN
NAN / 0 => NAN
Infinity / 0 => Infinity
-Infinity / 0 => -Infinity
```

Integer division has no concept of NAN / Infinity and so NULL and ERROR are the only two possible options.

Reply:
oh, i was more of wondering why fp64/fp64 didn't have the `IEEE` option.

Reply:
Oh!  Good catch :no_mouth:

Reply:
Fixed.